src/Notification.ts

```diff
@@ -11,11 +11,11 @@ export class Notification<T> {
   observe(observer: Observer<T>): any {
```

Comment:
This https://github.com/Microsoft/TypeScript/issues/4889 might be relevant to what we want?